pyproject.toml

```diff
@@ -5,10 +5,15 @@ exclude = [
     "setup.py",
 ]
 line-length = 79
-lint.ignore = []
+lint.ignore = [
+    "PT011", # use match in pytest.raises
```

Comment:
Why not include this?

Reply:
Surely it would make sense. I think I was just too lazy to add it manually at any time.